core/src/main/scala/com/github/johnynek/paiges/Doc.scala

```diff
@@ -823,4 +838,19 @@ object Doc {
    */
   def stack(ds: Iterable[Doc]): Doc =
     intercalate(line, ds)
+
+  /**
+   * build a table with the strings left aligned and
+   * the Docs starting in the column after the longest string.
+   * The Docs on the right are rendered aligned after the rightSep
+   */
+  def tabulate(leftSep: String, fill: Char, rightSep: String, kv: List[(String, Doc)]): Doc =
```

Comment:
@coltfred Presumably `leftSep` could be a fixed amount of spaces, rather than a single space? (That said, most indentation should usually be handled with `.nest` calls, meaning `leftSep` might be often unnecessary.)